### PR TITLE
[euscollada] Add documentation for predefined poses

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

**Comment by wkentaro** [comment]
```
% cat pr2.l | grep '(:reset-manip-pose' -A8
    (:reset-manip-pose
      (&optional (limbs '(:torso :larm :rarm :head)))
      """Predefined pose named reset-manip-pose."""
      (unless (listp limbs) (setq limbs (list limbs)))
      (dolist (limb limbs)
        (case limb
          (:torso (send self limb :angle-vector (float-vector 300.000000)))
          (:larm (send self limb :angle-vector (float-vector 75.000000 50.000000 110.000000 -110.000000 -20.000000 -10.000000 -10.000000)))
          (:rarm (send self limb :angle-vector (float-vector -75.000000 50.000000 -110.000000 -110.000000 20.000000 -10.000000 -10.000000)))
```

**Comment by k-okada** [comment]
based on @snozawa 's request, https://github.com/jsk-ros-pkg/jsk_model_tools/pull/172#issuecomment-209165271

**Comment by k-okada** [comment]
please rebase origin/master

**Comment by wkentaro** [comment]
Rebased.

File: euscollada/src/collada2eus.cpp [hunk 1]
```diff
@@ -1604,10 +1604,12 @@ int main(int argc, char* argv[]){
       }
       fprintf(output_fp,
               "\n"
-              "    (:%s (&optional (limbs '(%s)))\n"
+              "    (:%s\n"
+              "      (&optional (limbs '(%s)))\n"
+              "      \"\"\"Predefined pose named %s.\"\"\"\n"
```
Comment (k-okada) [inline euscollada/src/collada2eus.cpp]: @wkentaro it should be `"`.
```
(defun hoge ()
   """hoge"""
  (+ 1 2))
```
is considered as
```
(defun hoge ()
  "" ;; <- documentation string
  "hoge" ;; <-- eval hoge string
  "" ;; <- eval null string
  (+ 1 2)) ;; returns 3
```

Found by @YoheiKakiuchi

Comment (wkentaro) [inline euscollada/src/collada2eus.cpp]: Sorry, so it should be
```
\"Predefined pose named %s.\"\n"
```
?

**Comment by k-okada** [comment]
https://github.com/jsk-ros-pkg/jsk_model_tools/pull/204/files

--
◉ Kei Okada


2017-04-04 22:16 GMT+09:00 Kentaro Wada <notifications@github.com>:

**Pull request merged**